### PR TITLE
Added getter for version name

### PR DESCRIPTION
**Pull request by Raibaz**

### CODE REVIEW

**Comment by nwbirnie** [comment]
LGTM, do you want to also include updating the visibility modifier on ApiCatalog.getDefault()?

**Comment by Raibaz** [comment]
Don't think it's needed, the ApiCatalog interface is public so its methods are public by default.

File: google-ads/src/main/java/com/google/ads/googleads/lib/catalog/Version.java [hunk 1]
```diff
@@ -50,6 +50,14 @@
         versionName.matches("v[0-9]+"), "Invalid version name: %", versionName);
   }
 
+  /**
+   * Returns the version name, e.g. "v1".
+   * @return the version name
+   */
+  public String getVersionName() {
```
Comment (jradcliff) [inline google-ads/src/main/java/com/google/ads/googleads/lib/catalog/Version.java]: Could you elaborate on the motivation behind this change?

Comment (Raibaz) [reply google-ads/src/main/java/com/google/ads/googleads/lib/catalog/Version.java]: Accessing the name (i.e. the number) of the latest version programmatically may be useful for client applications that want to use reflection to instantiate classes like resources or enums, like an entity class generator that generates a class for each resource and wants to use the appropriate class for the enum fields.

**Comment by nwbirnie** [comment]
LGTM

**Pull request merged**